ambry-clustermap/src/main/java/com.github.ambry.clustermap/ClusterMapUtils.java

```diff
@@ -130,7 +143,8 @@ public static String getInstanceName(String host, Integer port) {
    * @return the rack id associated with the given instance.
    */
   static Long getRackId(InstanceConfig instanceConfig) {
```

Comment:
Is @cgtz's  change turning this into a `String`?

Reply:
Yes #895 will change this to return the string directly instead of converting it into a long.

Reply:
Thanks. Will address this based on what gets merged first.